### DISCUSSION TITLE
can anyone tell me what this file for solidstart will look like, I plan to use solidstart for the external API

### DISCUSSION THREAD

**Discussion by sh1man999** [solidjs/solid-start#1334] [Q&A]
This example SPA:
https://github.com/solidjs/solid-realworld/blob/main/src/App.js
I need example Solidstart

**Comment by doeixd** [comment]
Hello! Thanks for your interest in SolidStart. The project you linked to is not actually using SolidStart. If you would like to use something to create an API, could I recommend you take a look at: nitro.unjs.io

**Comment by sh1man999** [comment]
Can someone show me an example of how solid start works with microservice architecture?
